src/lib/xmlParse/xmlQueryContextResponse.cpp

```diff
@@ -374,6 +374,5 @@ void qcrsPresent(ParseData* parseDataP)
   if (!lmTraceIsSet(LmtDump))
     return;
 
```

Comment:
In the case it gets printed... Is it using the DEBUG level? (https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide#Log_format)